templates/full_template.html

```diff
@@ -0,0 +1,481 @@
+<!DOCTYPE html>
```

Comment:
why does this file exist? what is it?

Reply:
This is the file that Shein created as an example template of what the printlist would look like (without JS integration). Should I remove the file, now that JS has been integrated?

Reply:
I think so